pkg/router/template/router.go

```diff
@@ -167,6 +173,31 @@ func newTemplateRouter(cfg templateRouterCfg) (*templateRouter, error) {
 	return router, nil
 }
 
+func isInteger(s string) bool {
+	_, err := strconv.Atoi(s)
+	return (err == nil)
+}
+
+func matchValues(s string, allowedValues ...string) bool {
+	for _, value := range allowedValues {
+		if value == s {
+			return true
+		}
+	}
+	return false
+}
+
+func matchString(pattern, s string) bool {
```

Comment:
In https://github.com/openshift/origin/pull/9671 changing matchString to matchPattern in order to change default behavior to anchor on the beginning and end of the line. To avoid confusion with regexp.MatchString() the name was changed